src/data/constants/app.js

```diff
@@ -57,12 +57,12 @@ export const bulkManagementColumns = [
 export const messages = StrictDict({
   BulkManagementTab: StrictDict({
     csvUploadLabel: 'Upload Grade CSV',
-    heading: 'Use this feature by downloading a CSV for bulk management, overriding grades locally and coming back here to upload',
+    heading: 'Use this feature by downloading a CSV for bulk management, overriding grades locally and coming back here to upload.',
```

Comment:
Same note here about oxford comma

```suggestion
    heading: 'Use this feature by downloading a CSV for bulk management, overriding grades locally, and coming back here to upload.',
```